crt_portal/cts_forms/migrations/0067_populate_new_choice_fields.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 2.2.12 on 2020-04-30 18:00
+
+from django.db import migrations
+
+from cts_forms.model_variables import (HATE_CRIMES_TRAFFICKING_MODEL_CHOICES,
+                                       PROTECTED_MODEL_CHOICES)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cts_forms', '0066_add_new_choicefields'),
+    ]
+
+    def forward(apps, schema_editor):
+        """
+        Set value field of existing HateCrimesandTrafficking and ProtectedClass instances
+        to those matching their labels set in original field
+
+        This migration is expected to fail and no changes saved if _any_
+        existing HateCrimesandTrafficking or ProtectedClass instance cannot
+        be updated
+        """
+        HateCrimesandTrafficking = apps.get_model('cts_forms', 'HateCrimesandTrafficking')
+        ProtectedClass = apps.get_model('cts_forms', 'ProtectedClass')
+
+        hc_labels = {label: value for value, label in HATE_CRIMES_TRAFFICKING_MODEL_CHOICES}
+
+        for hc in HateCrimesandTrafficking.objects.all():
+            try:
+                hc.value = hc_labels[hc.hatecrimes_trafficking_option]
+            except KeyError:
+                # Live instances may have punctuation, try to match without
+                hc.value = hc_labels[hc.hatecrimes_trafficking_option[:-1]]
+            hc.save()
+
+        pc_labels = {label: value for value, label in PROTECTED_MODEL_CHOICES}
+        for pc in ProtectedClass.objects.all():
+            pc.value = pc_labels[pc.protected_class]
+            pc.save()
+
+
+    def reverse(apps, schema_editor):
+        """Empty the values places in `value`"""
+        HateCrimesandTrafficking = apps.get_model('cts_forms', 'HateCrimesandTrafficking')
+        ProtectedClass = apps.get_model('cts_forms', 'ProtectedClass')
+        for hc in HateCrimesandTrafficking.objects.all():
+            hc.value = ''
+            hc.save
+
+        for pc in ProtectedClass.objects.all():
```

Comment:
OK, one more thing. Can we loop though the protected class options instead of the existing class objects 

We used to aggregate the data in protected class differently. So there are going to be more protected classes than we have protected class option. For example, we used to have 

 - National origin (including ancestry, ethnicity, and language)

Then, that was split into two groups. 
 - National origin (including ancestry and ethnicity)
 - Language

This was our first test case on making sure the data could evolve. It is used as the example in the protected class changes in the infrequent tasks and maintenance docs: https://github.com/usdoj-crt/crt-portal/blob/develop/docs/maintenance_or_infrequent_tasks.md#change-protected-class-options.

I don't think this particular issue happens in Production since we only added data there recently, but that list is going to change over time and we want a way to preserve past data and update display when those things don't have one to one matches.